### PR TITLE
remove indications that are not used anymore

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A  |
|---          |---|
|Bug Fix?     |n  |
|New Feature? |n  |
|BC Breaks?   |n  |
|Deprecations?|n  |
|Tests Pass?  |n  |
|Fixed Tickets|   |
|License      |MIT|
|Doc PR       |   |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,51 +40,9 @@ Exceptions, mostly down to PHP language barriers:
...
-  get_include_path()
-);
-
-require_once 'hamcrest/Hamcrest/MatcherAssert.php';
-require_once 'hamcrest/Hamcrest/Matchers.php';
-
-Hamcrest_MatcherAssert::assertThat('a', Hamcrest_Matchers::equalToIgnoringCase('A'));
-```
-
-Composer Package
```
Comment (aik099) [inline README.md]: Are you sure you want to remove Composer part as well?

Comment (cordoval) [inline README.md]: yes because it does not add any value and it is defacto standard

**Comment by aik099** [comment]
I don't like these "coverage increased/decreased" messages on PR's. @cordoval , can you please disable them in settings on Coveralls.Io website?

**Comment by cordoval** [comment]
done

**Pull request merged**